nimx/view.nim

```diff
@@ -210,11 +210,21 @@ proc insertSubview*(v, s: View, i: int) =
         s.viewDidMoveToWindow()
         v.didAddSubview(s)
         v.setNeedsDisplay()
+    else:
```

Comment:
This logic is too complex. You just need to change view position in the `subviews` array. No need to call `removeFromSuperview` and `didAddSubview`